tests/bud.bats

```diff
@@ -5258,6 +5258,37 @@ _EOF
     assert "$output" =~ ".*accepts at most 1 arg\(s\), received 2" "Should fail when passed extra arg after context directory"
 }
 
+@test "bud with --no-hostname" {
+  skip_if_no_runtime
+
+  _prefetch alpine
+
+  mytmpdir=${TEST_SCRATCH_DIR}/my-dir
+  mkdir -p ${mytmpdir}
+  cat > $mytmpdir/Containerfile << _EOF
+from alpine
+run cat /etc/hostname
+_EOF
+
+  run_buildah build --no-cache -t testbud \
+                  $WITH_POLICY_JSON --file ${mytmpdir}/Containerfile .
+  assert "${lines[2]}" != "localhost" "Should be set to something other then localhost"
+
+  run_buildah build --no-hostname --no-cache -t testbud \
+                  $WITH_POLICY_JSON --file ${mytmpdir}/Containerfile .
+  assert "${lines[2]}" == "localhost" "Should be set to localhost"
+
+  cat > $mytmpdir/Containerfile << _EOF
+from alpine
+RUN mv /etc /usr/
+RUN ls -l /etc
```

Comment:
The content of this file will always be the same.  Why are we generating it every time this test runs?

Reply:
This file has different content then the previous?

Reply:
Use two static files?